src/components/menu/menu.spec.ts

```diff
@@ -376,6 +383,38 @@ describe('MenuComponent', () => {
 
         expect(component.item1Activated).toHaveBeenCalledTimes(1);
     });
+
+    it('should not emit a click event when disabled is true.', async () => {
+        component.disabled = true;
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        // open menu
+        component.trigger.openMenu();
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        // get the child submenu item
+        const menuItem = document.querySelector<HTMLButtonElement>(
+            '#menu-item-disabled'
+        );
+        expect(menuItem).toBeTruthy();
+
+        const item1Element = overlayContainerElement.querySelector('#menu-item-1') as HTMLButtonElement;
```

Comment:
Perhaps we could name this something like `itemWithSubMenu` or something like this to indicate the difference between the two?